prefect_dbt/cli/configs/base.py

```diff
@@ -78,7 +81,25 @@ def get_configs(self) -> Dict[str, Any]:
         return self._populate_configs_json({}, self.__fields__, model=self)
 
 
-class TargetConfigs(DbtConfigs):
+class AbstractTargetConfigs(DbtConfigs, abc.ABC):
```

Comment:
Small thing: I'd call this `BaseTargetConfigs` since it doesn't have any abstract methods.
```suggestion
class BaseTargetConfigs(DbtConfigs, abc.ABC):
```